include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -276,6 +276,7 @@ class alignment_algorithm :
     void initialise_matrix(cache_t & cache)
     {
         auto col = this->current_column();
+        auto score_col = col | view_get_score_column;
```

Comment:
Doesn't the traceback matrix need to be initialized too? (what's the default btw?)

Reply:
The traceback matrix will be written to within the computation.